.github/workflows/go.yml

```diff
@@ -3,9 +3,11 @@ on:
   push:
     branches:
       - master
+      - feature/**
```

Comment:
You'll need to drop this. We can't actually push the Go branch on pushes to feature branches :)

Or, you'll need to update the "if" below and check that the branch really is the master branch.

Reply:
This pr was merged, so I create a new one:
https://github.com/google/gvisor/pull/3886